src/main/java/co/inventorsoft/academy/service/MainService.java

```diff
@@ -20,8 +20,8 @@ public void mainScreen() {
         System.out.println("\t\t\t\t\tPlease, register your team to derby with Formula1 teams in 1vs1 format");
         System.out.println(RESET + "************************************************************" +
                 "********************************************************");
-        registration.createTeam();
-        tournament.start();
+        registration.createTeam(); //registration service
```

Comment:
Try to not use this sort of comment. You can describe your method in method docs. /**  */